variables.tf

```diff
@@ -159,6 +159,7 @@ variable "materialized_views" {
       }),
     }),
     expiration_time = string,
+    max_staleness   = optional(string),
```

Comment:
Since we are using optional, could you also update min TF version to >= 1.3

Reply:
this is done, funny that we have skipped this breaking change for that long.